<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Agregar Libro</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leNombre">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Inserta el nombre</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leAutor">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Inserta el autor</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leGenero">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Inserta el genero</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAgregar">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>360</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
